code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -38,8 +38,8 @@
 	icon_state = "mouse_[body_color]"
 	icon_living = "mouse_[body_color]"
 	icon_dead = "mouse_[body_color]_dead"
-	if(prob(40))
-		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(2, 4))
+	if(prob(75))
+		var/datum/disease/advance/R = new /datum/disease/advance/random(rand(3, 6), rand(4, 9))
```

Comment:
ditto

Reply:
Ah shit did I get the symptom count and symptom level backwards

Reply:
Yeah

Reply:
Should be fixed now.

Reply:
Actually I am changing up the random disease generation a tad

By Default it will have a minimum of level 1
But you can pass an arg to skew towards specific levels